### PR TITLE
fix bug about label_map in evaluation part

### PR DESCRIPTION
**Pull request by rlatjcj**
## Motivation

According to https://github.com/open-mmlab/mmsegmentation/pull/1445, evaluation part should be changed, too.

## Modification

Record the initial label with `.clone()`, because `label` is `torch.tensor`.

### CODE REVIEW

**Review by MeowZheng — APPROVED** [review]
Thanks for the bug fix, it looks good to me. Please sign CLA first
